ts/src/bybit.ts

```diff
@@ -2737,15 +2664,29 @@ export default class bybit extends Exchange {
         const feeCostString = this.safeString (trade, 'execFee');
         let fee = undefined;
         if (feeCostString !== undefined) {
+            const feeRateString = this.safeString (trade, 'feeRate');
             let feeCurrencyCode = undefined;
             if (market['spot']) {
-                feeCurrencyCode = this.safeString (trade, 'commissionAsset');
+                if (Precise.stringGt (feeCostString, '0')) {
```

Comment:
@ndubel I think this logic produces inversed values for the fee currency, check this out:

```

bybit.fetchMyTrades(BTC/USDT,None,2)
[{'amount': 0.000331,
  'cost': 9.9787563,
  'datetime': '2023-11-09T15:02:54.077Z',
  'fee': {'cost': 3.31e-07, 'currency': 'BTC', 'rate': 0.001},
  'fees': [{'cost': 3.31e-07, 'currency': 'BTC', 'rate': 0.001}],
  'id': '2100000000046970325',
  'info': {'blockTradeId': '',
           'closedSize': '',
           'execFee': '0.000000331',
           'execId': '2100000000046970325',
           'execPrice': '30147.3',
           'execQty': '0.000331',
           'execTime': '1699542174077',
           'execType': 'Trade',
           'execValue': '9.9787563',
           'feeRate': '0.001',
           'indexPrice': '',
           'isMaker': False,
           'leavesQty': '0',
           'markIv': '',
           'markPrice': '',
           'orderId': '1550266550311982592',
           'orderLinkId': '1550266550311982593',
           'orderPrice': '999999',
           'orderQty': '0',
           'orderType': 'Market',
           'seq': '1251220581',
           'side': 'Buy',
           'stopOrderType': '',
           'symbol': 'BTCUSDT',
           'tradeIv': '',
           'underlyingPrice': ''},
  'order': '1550266550311982592',
  'price': 30147.3,
  'side': 'buy',
  'symbol': 'BTC/USDT',
  'takerOrMaker': 'taker',
  'timestamp': 1699542174077,
  'type': 'market'},
 {'amount': 0.0001,
  'cost': 3.016195,
  'datetime': '2023-11-09T15:07:13.907Z',
  'fee': {'cost': 0.003016195, 'currency': 'USDT', 'rate': 0.001},
  'fees': [{'cost': 0.003016195, 'currency': 'USDT', 'rate': 0.001}],
  'id': '2100000000046971118',
  'info': {'blockTradeId': '',
           'closedSize': '',
           'execFee': '0.003016195',
           'execId': '2100000000046971118',
           'execPrice': '30161.95',
           'execQty': '0.0001',
           'execTime': '1699542433907',
           'execType': 'Trade',
           'execValue': '3.016195',
           'feeRate': '0.001',
           'indexPrice': '',
           'isMaker': False,
           'leavesQty': '0',
           'markIv': '',
           'markPrice': '',
           'nextPageCursor': '4093397%3A0%2C4093139%3A0',
           'orderId': '1550268729923999232',
           'orderLinkId': '1550268729923999233',
           'orderPrice': '0.01',
           'orderQty': '0.0001',
           'orderType': 'Market',
           'seq': '1251226555',
           'side': 'Sell',
           'stopOrderType': '',
           'symbol': 'BTCUSDT',
           'tradeIv': '',
           'underlyingPrice': ''},
  'order': '1550268729923999232',
  'price': 30161.95,
  'side': 'sell',
  'symbol': 'BTC/USDT',
  'takerOrMaker': 'taker',
  'timestamp': 1699542433907,
  'type': 'market'}]
```
it is saying that for my market-buy order the fee is in Base and for the market sell is in the quote currency

Reply:
for both of them the feeRate is positive

Reply:
and what the problem? In your orders it's all ok with fee currency. In buy order {'cost': 3.31e-07, 'currency': 'BTC'} so very small amount in BTC, in second sell order {'cost': 0.003016195, 'currency': 'USDT'}, also right

Reply:
@ndubel I'm sorry you're right, I confused myself because I'm used to seeing the opposite (when buying the fee is deducted using the quote and vice-versa) but that's not what happens here: 

![image](https://github.com/ccxt/ccxt/assets/43336371/1c6517b5-f4c7-4cbd-a144-6df7605944bb)


Will be merged soon, sorry for the confusion!